go/vt/vtgate/evalengine/testcases/cases.go

```diff
@@ -29,151 +29,151 @@ import (
-	{Run: FnIsIPv4Mapped},
-	{Run: FnIsIPv6},
-	{Run: FnBinToUUID},
-	{Run: FnIsUUID},
-	{Run: FnUUID},
-	{Run: FnUUIDToBin},
-	{Run: DateMath},
-	{Run: RegexpLike},
-	{Run: RegexpInstr},
-	{Run: RegexpSubstr},
-	{Run: RegexpReplace},
+	// {Run: StrcmpComparison},
+	// {Run: MultiComparisons},
+	// {Run: IntervalStatement},
+	// {Run: IsStatement},
+	// {Run: NotStatement},
+	// {Run: LogicalStatement},
+	// {Run: TupleComparisons},
+	// {Run: Comparisons},
+	// {Run: InStatement},
+	// {Run: FnField},
+	// {Run: FnElt},
+	// {Run: FnInsert},
+	// {Run: FnLower},
+	// {Run: FnUpper},
+	// {Run: FnCharLength},
+	// {Run: FnLength},
+	// {Run: FnBitLength},
+	// {Run: FnAscii},
+	// {Run: FnReverse},
+	// {Run: FnSpace},
+	// {Run: FnOrd},
+	// {Run: FnRepeat},
+	// {Run: FnLeft},
+	// {Run: FnLpad},
+	// {Run: FnRight},
+	// {Run: FnRpad},
+	// {Run: FnLTrim},
+	// {Run: FnRTrim},
+	// {Run: FnTrim},
+	// {Run: FnSubstr},
+	// {Run: FnLocate},
+	// {Run: FnReplace},
+	// {Run: FnConcat},
+	// {Run: FnConcatWs},
+	// {Run: FnChar},
+	// {Run: FnHex},
+	// {Run: FnUnhex},
+	// {Run: FnCeil},
+	// {Run: FnFloor},
+	// {Run: FnAbs},
+	// {Run: FnPi},
+	// {Run: FnAcos},
+	// {Run: FnAsin},
+	// {Run: FnAtan},
+	// {Run: FnAtan2},
+	// {Run: FnCos},
+	// {Run: FnCot},
+	// {Run: FnSin},
+	// {Run: FnTan},
+	// {Run: FnDegrees},
+	// {Run: FnRadians},
+	// {Run: FnNow, Compare: &Comparison{LooseTime: true}},
+	// {Run: FnInfo},
+	// {Run: FnExp},
+	// {Run: FnLn},
+	// {Run: FnLog},
+	// {Run: FnLog10},
+	// {Run: FnMod},
+	// {Run: FnLog2},
+	// {Run: FnPow},
+	// {Run: FnSign},
+	// {Run: FnSqrt},
+	// {Run: FnRound},
+	// {Run: FnTruncate},
+	// {Run: FnCrc32},
+	// {Run: FnConv},
+	// {Run: FnBin},
+	// {Run: FnOct},
+	// {Run: FnMD5},
+	// {Run: FnSHA1},
+	// {Run: FnSHA2},
+	// {Run: FnRandomBytes},
+	// {Run: FnDateFormat},
+	// {Run: FnConvertTz},
+	// {Run: FnDate},
+	// {Run: FnDayOfMonth},
+	// {Run: FnDayOfWeek},
+	// {Run: FnDayOfYear},
+	// {Run: FnFromUnixtime},
+	// {Run: FnHour},
+	// {Run: FnMakedate},
+	// {Run: FnMaketime},
+	// {Run: FnMicroSecond},
+	// {Run: FnMinute},
+	// {Run: FnMonth},
+	// {Run: FnMonthName},
+	// {Run: FnLastDay},
+	// {Run: FnToDays},
+	// {Run: FnFromDays},
+	// {Run: FnSecToTime},
+	// {Run: FnTimeToSec},
+	// {Run: FnToSeconds},
+	// {Run: FnQuarter},
+	// {Run: FnSecond},
+	// {Run: FnTime},
+	// {Run: FnUnixTimestamp},
+	// {Run: FnWeek},
+	// {Run: FnWeekDay},
+	// {Run: FnWeekOfYear},
+	// {Run: FnYear},
+	// {Run: FnYearWeek},
+	// {Run: FnPeriodAdd},
+	// {Run: FnPeriodDiff},
+	// {Run: FnInetAton},
+	// {Run: FnInetNtoa},
+	// {Run: FnInet6Aton},
+	// {Run: FnInet6Ntoa},
+	// {Run: FnIsIPv4},
+	// {Run: FnIsIPv4Compat},
+	// {Run: FnIsIPv4Mapped},
+	// {Run: FnIsIPv6},
+	// {Run: FnBinToUUID},
+	// {Run: FnIsUUID},
+	// {Run: FnUUID},
+	// {Run: FnUUIDToBin},
+	// {Run: DateMath},
+	// {Run: RegexpLike},
+	// {Run: RegexpInstr},
+	// {Run: RegexpSubstr},
+	// {Run: RegexpReplace},
```

Comment:
@systay Did you intend to disable all these tests?

Reply:
oops